### DISCUSSION TITLE
Github repositories are not updating

### DISCUSSION THREAD

**Discussion by yahu1031** [community/community#3981] [Mobile] [answered]
I used to have 96 repositories in my account. As I don't have delete repo option in mobile I deleted from website. After when I go to repositories in mobile to still shows the old repositories. IDK why. I cleared the data of app and logged-in back again but still same. Hope I will get an answers if we have, else please add it to issues. Even API endpoint has correct no.of repos. This is in home repositories. Are they the repos where i have made something???

**Comment by elisealix22** [comment, marked as answer]
Thank you for reaching out! The repositories list from Home is populated with repositories that you are a collaborator on. If you remove yourself as a collaborator for the repository, it should be removed from the list on Home. Please let me know if this doesn't work for you! Also if you have any more requests or bugs for our apps, please reach back out. We'd love to hear them.

**Comment by yahu1031** [comment]
Hello @elisealix22,

Yes that perfectly works and I realized just now. And one more suggestion is about **Bottom Navigation Bar icons**. When we tab on it there is splash happening and it was really bad. I mean that looks weird. Can you please disable(splash color to transparent or something) it. it looks pretty cool then. And according to UI, the enabled icon are filled. but notifications icon and profile icons weren't. Better we have any other documents viewing libraries added like PDF readers/ docs readers cause they were going out to webview and downloading stuff. We also better have settings option for our(user's) repositories so that we can handle from the bile itself. If you are working on it already then good luck else add it please.

Regards

**Comment by zawzawaung-prog** [comment]
> Thank you for reaching out! The repositories list from Home is populated with repositories that you are a collaborator on. If you remove yourself as a collaborator for the repository, it should be removed from the list on Home. Please let me know if this doesn't work for you! Also if you have any more requests or bugs for our apps, please reach back out. We'd love to hear them.

[]()
